tf2onnx/onnx_opset/tensor.py

```diff
@@ -1464,18 +1465,41 @@ def version_10(cls, ctx, node, **kwargs):
         # onnx output is [num_selected_boxes, 3], the meaning of last dim is [batch_index, class_index, box_index]
         # while tf's output is [num_selected_boxes]
         ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[0], axes=[0])
-        ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], axes=[0, 1])
+        input_score = ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], axes=[0, 1])
         ctx.insert_new_node_on_input(node, "Cast", node.input[2], to=onnx_pb.TensorProto.INT64)
         # replace original node with nonmaxsurppress + slice + squeeze + cast
-        dtypes = [ctx.get_dtype(node.output[0])]
-        shapes = [ctx.get_shape(node.output[0])]
+        dtypes = [[ctx.get_dtype(output)] for output in node.output]
+        shapes = [[ctx.get_shape(output)] for output in node.output]
+        max_output_size = node.input[2]
+        utils.make_sure(len(node.inputs) <= 5 or int(node.inputs[5].get_tensor_value(False)) == 0,
+                        "soft_nms_sigma must be 0")
```

Comment:
is node.inputs[5] guaranteed to be const? If not, should check that it's const before getting tensor value.
If it's not const, should throw exception.

Reply:
get_tensor_value throws exception on non-const case in the very beginning.